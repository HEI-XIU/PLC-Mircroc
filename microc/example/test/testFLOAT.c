void main()
{
    float h;
    float s;
    float sum;
    h = 1.25;
    // s = 1.25+1;
    // sum =h+s;
    print h ;
    // printf("%f",h);
    // printf("%f",sum);
    
}