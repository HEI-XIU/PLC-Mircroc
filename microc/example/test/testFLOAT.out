24 19 0 5 25 15 1 15 1 15 1 13 0 0 1 0 1067450368 12 15 -1 13 0 0 1 11 22 15 -1 15 -3 21 -1